pages/data-providers/[data-provider-id]/deposit-dates.jsx

```diff
@@ -8,6 +9,7 @@ import TimeLagChart from 'components/time-lag-chart'
 import { Button, Card, Label } from 'design'
 import * as texts from 'texts/depositing'
 import DocumentLink from 'components/document-link'
+import { API_URL } from 'config'
```

Comment:
I would like to avoid URL concatenation here. Can you add a pre-calculated property into the store and use it from there?

Reply:
https://github.com/oacore/dashboard/pull/190/commits/049212e282c84a297c15575d02cc4ab911648a73